src/main/scala/uk/gov/nationalarchives/notifications/decoders/ExportStatusDecoder.scala

```diff
@@ -2,15 +2,19 @@ package uk.gov.nationalarchives.notifications.decoders
 
 import java.util.UUID
 
-import com.amazonaws.services.lambda.runtime.events.SNSEvent
 import io.circe.CursorOp.DownField
-import io.circe.{Decoder, HCursor}
-import io.circe._
-import io.circe.parser._
 import io.circe.generic.auto._
+import io.circe.parser._
+import io.circe.{Decoder, HCursor, _}
 
 object ExportStatusDecoder {
-  case class ExportStatusEvent(consignmentId: UUID, success: Boolean, environment: String) extends IncomingEvent
+  case class ExportSuccessDetails(userId: UUID, consignmentReference: String, transferringBodyCode: String)
+  case class ExportStatusEvent(
+                                consignmentId: UUID,
+                                success: Boolean,
+                                environment: String,
+                                successDetails: Option[ExportSuccessDetails],
+                                failureCause: Option[String]) extends IncomingEvent
```

Comment:
Is this just temporarily optional? I remember we talked about making an error message non-optional last week, but I can't remember if it was this one?

Reply:
This would be optional. Either the export is successful, in which case the success details would be available, via the "publishSuccess". Or the export is unsuccessful, in which case the cause would be available via the "publishFailure"